repos/package-source-build.proj

```diff
@@ -24,7 +24,9 @@
     <ItemGroup>
       <CoreClrToolsFiles Include="$(ToolPackageExtractDir)coreclr-tools/*" />
     </ItemGroup>
+    <!-- Also copy to the legacy path without the architecture for SBRP for now -->
     <Copy SourceFiles="@(CoreClrToolsFiles)" DestinationFolder="$(SourceBuiltPackagesPath)coreclr-tools" />
+    <Copy SourceFiles="@(CoreClrToolsFiles)" DestinationFolder="$(SourceBuiltPackagesPath)coreclr-tools/$(BuildArchitecture)" />
```

Comment:
I wonder if it would be better to always use the architecture-less path.

When we use `/usr/bin/clang` we dont worry whether that's for `x86-64` or `aarch64`, we just assume it's for *this* architecture. Same for `/usr/bin/dotnet`, we don't check what architecture that's for. We just use it.

Along the same line, shouldn't we assume `$(SourceBuiltPackagesPath)coreclr-tools` is the right set of tools for this architecture?